lxd/db/warnings_types.go

```diff
@@ -80,6 +82,7 @@ var WarningTypeNames = map[WarningType]string{
 	WarningNetworkStartupFailure:                  "Failed to start network",
 	WarningOfflineClusterMember:                   "Offline cluster member",
 	WarningInstanceAutostartFailure:               "Failed to autostart instance",
+	WarningInstanceTypeNotOperational:             "Instance type not operational",
```

Comment:
How about using "not supported" to align with the `SupportedInstanceTypes` function name?

Reply:
Oh no ignore me, I just saw the error message from the original issue says "operational", this is fine as-is.